logging/cluster-logging-release-notes.adoc

```diff
@@ -8,34 +8,65 @@ toc::[]
+
+* TBD-description
+(link:https://issues.redhat.com/browse/LOG-TBD[LOG-TBD])
+
+* With this update, if you use the Fluentd forward protocol to forward log data over a TLS-encrypted connection, you can now use a password-encrypted private key file and specify the passphrase in the Cluster Log Forwarder configuration. For more information, see xref:../logging/cluster-logging-external.html#cluster-logging-collector-log-forward-fluentd_cluster-logging-external[Forwarding logs using the Fluentd forward protocol]
+(link:https://issues.redhat.com/browse/LOG-1525[LOG-1525])
+
+* With this update, you can use a username and password to authenticate a log forwarding connection to an external Elasticsearch instance. For example, if you cannot use mutual TLS (mTLS) because a third-party operates the Elasticsearch instance, you can use HTTP or HTTPS and set a secret that contains the username and password. For more information see xref:../logging/cluster-logging-external.adoc#cluster-logging-collector-log-forward-es_cluster-logging-external[Forwarding logs to an external Elasticsearch instance].
+(link:https://issues.redhat.com/browse/LOG-974[LOG-1022])
+
+
+
+[id="openshift-logging-5-2-0-deprecated-removed-features"]
+=== Deprecated and removed features
+
+Some features available in previous releases have been deprecated or removed.
+
+Deprecated functionality is still included in OpenShift Logging and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments.
+
+[id="openshift-logging-5-2-0-tbd"]
+==== TBD-title
+
+TBD-description
+
+[id="openshift-logging-5-2-0-bug-fixes"]
+=== Bug fixes
+
+* TBD-description
+(link:https://issues.redhat.com/browse/LOG-TBD[LOG-TBD])
+
+* Before this update, if you redeployed a full Elasticsearch cluster, it got stuck in an unhealthy state, with one non-data node running and all other data nodes shut down. This happened because new certificates prevented the Elasticsearch Operator from scaling down the non-data nodes of the Elasticsearch cluster. With this update, Elasticsearch Operator can scale all the data and non-data nodes down and then back up again, so they load the new certificates. The Elasticsearch Operator can reach the new nodes after they load the new certificates.
+(link:https://issues.redhat.com/browse/LOG-1524[LOG-1524])
+
+* Before this update, if you redeployed a full Elasticsearch cluster, it got stuck in an unhealthy state, with one non-data node running and all other data nodes shut down. This happened because new certificates prevented the Elasticsearch Operator from scaling down the non-data nodes of the Elasticsearch cluster. With this update, Elasticsearch Operator can scale all the data and non-data nodes down and then back up again, so they load the new certificates. The Elasticsearch Operator can reach the new nodes after they load the new certificates.
+(link:https://issues.redhat.com/browse/LOG-1537[LOG-1537])
+
```

Comment:
Looks like a duplicate of the previous point?

Reply:
Removed. Thanks!